bin/streamr-stream-grant-permission.ts

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node -r ts-node/register
+import { Command } from 'commander';
+import {
+    envOptions,
+    authOptions,
+    formStreamrOptionsWithEnv
+} from './common'
+import pkg from '../package.json'
+import { AnonymousStreamPermisson, StreamOperation, StreamrClient, UserStreamPermission } from 'streamr-client';
+import EasyTable from 'easy-table'
+
+const PUBLIC_PERMISSION_ID = 'public'
+
+const OPERATIONS: Record<string,StreamOperation> = {
+    'get': StreamOperation.STREAM_GET,
+    'edit': StreamOperation.STREAM_EDIT,
+    'delete': StreamOperation.STREAM_DELETE,
+    'publish': StreamOperation.STREAM_PUBLISH,
+    'subscribe': StreamOperation.STREAM_SUBSCRIBE,
+    'share': StreamOperation.STREAM_SHARE
+}
+
+const getOperation = (id: string) => {
+    const operation = OPERATIONS[id]
+    if (operation === undefined) {
```

Comment:
I think it would be nice if it also worked with the values `stream_get`, `stream_publish` etc. as defined in Core API and client.

Reply:
Implemented the support